src/main/java/nextstep/courses/domain/Money.java

```diff
@@ -0,0 +1,49 @@
+package nextstep.courses.domain;
+
+import java.util.Objects;
+
+public class Money {
+    private int money;
+
+    public Money(int money) {
+        validation(money);
+        this.money = money;
+    }
+
+    private void validation(int money) {
+        if (money < 0) {
+            throw new IllegalArgumentException("금액은 0 이상이어야 합니다.");
+        }
+    }
+
+    public boolean matches(int money) {
+        return this.money == money;
+    }
+
+    public boolean matches(Money money) {
+        return this.money == money.getMoney();
```

Comment:
```suggestion
        return matches(money.money);
```
오버로딩 메서드의 경우 다른 메서드를 호출해 구현함으로써 중복을 제거할 수 있음.